tslint.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": ["tslint:latest", "tslint-react"],
   "rules": {
+    "no-implicit-dependencies": [true, "dev"]
```

Comment:
I needed this to be able to import from dev dependencies like Enzyme. Is there a better solution?